docs/interactions/Receiving_and_Responding.mdx

```diff
@@ -376,7 +376,7 @@ Interaction tokens are valid for **15 minutes**, meaning you can respond to an i
 
 ## Create Interaction Response % POST /interactions/\{interaction.id#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object\}/\{interaction.token#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object\}/callback
 
-Create a response to an Interaction. Body is an [interaction response](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object). Returns `204` unless `with_response` is set to `true` which returns `200` with the body as  [interaction response](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object).
+Create a response to an Interaction. Body is an [interaction response](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object). Returns `204` unless `with_response` is set to `true` which returns `200` with the body as  [interaction response](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object).
```

Comment:
```suggestion
Create a response to an Interaction. Body is an [interaction response](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object). Returns `204` unless `with_response` is set to `true` which returns `200` with the body as [interaction response](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object).
```

Also a space too much

Reply:
Ok. I also just realized that it should be `Interaction Callback Response` instead of `Interaction Response`